docs/advisors/checks/mongodb-psa-architecture.md

```diff
@@ -0,0 +1,36 @@
+# MongoDB PSA Architecture
+
+## Description
+This check returns an error if the replicaSet is using a PSA architecture.
+
+## Resolution
+
+A replica set can be configured with either a PSS (Primary-Secondary-Secondary) or a PSA (Primary-Secondary-Arbiter) architecture.
+
+In a three-member replica set with a primary-secondary-arbiter (PSA) architecture or a sharded cluster with three-member PSA shards, the arbiter does not hold a copy of the data. These deployments provide only one complete copy of the data. They are used primarily for situations where nodes are located in different Data Centers and you want to preserve quorum in cases where you want to avoid split brain caused by potential network issues.
```

Comment:
```suggestion
In a three-member replica set with a primary-secondary-arbiter (PSA) architecture or a sharded cluster with three-member PSA shards, the arbiter does not hold a copy of the data. These deployments provide only one complete copy of the data. They are used primarily for situations where nodes are located in different data centers and you want to preserve quorum in cases where you want to avoid split brain caused by potential network issues.
```